library/src/tensor_operation_instance/gpu/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ add_subdirectory(batched_gemm)
 add_subdirectory(batched_gemm_reduce)
 add_subdirectory(batched_gemm_gemm)
 add_subdirectory(batched_gemm_softmax_gemm)
+add_subdirectory(batched_gemm_masking_scale_softmax_gemm_permute)
```

Comment:
I agree that w or w/o masking should use the same test structure. Could you do it in the next PR? Because for now, all the instances are dedicated for masking one.

Reply:
Sure thing!